internal/clientconn/conninfo/conn_info_test.go

```diff
@@ -72,3 +75,87 @@ func TestConnInfo(t *testing.T) {
 		})
 	}
 }
+
+func TestConnInfoCursorParallelWork(t *testing.T) {
```

Comment:
Let's add a comment to explain why exactly this test is needed and what we test here, this test doesn't have asserts, so it might make sense to explain why. Also, I think if it's possible, let's add some asserts to check data correctness.

Reply:
I added data to cursors and fill them with data that we assert when reading cursors.